src/bindings/python/src/pyopenvino/core/properties/properties.cpp

```diff
@@ -147,6 +147,22 @@ void regmodule_properties(py::module m) {
     wrap_property_RW(m_intel_gpu_hint, ov::intel_gpu::hint::host_task_priority, "host_task_priority");
     wrap_property_RW(m_intel_gpu_hint, ov::intel_gpu::hint::available_device_mem, "available_device_mem");
 
+    // Submodule intel_npu
+    py::module m_intel_npu =
```

Comment:
I've already added npu properties to python api: https://github.com/openvinotoolkit/openvino/pull/28358
please, just add the new one there.